values.yaml

```diff
@@ -28,6 +28,30 @@ ingress:
     # replace the variable reference if you want to use your own TLS secret for notary
     notarySecretName: *tls_secretName
 
+# Setting ingress.enabled to false will enable the service, and
+# the nginx will be deployed as the proxy
+service:
```

Comment:
The term `service` is quite confusing, it's not very clear that this is the service that points to the nginx pod if nginx pod is to be deployed.
Or we better clarify it in the comments?